### PR TITLE
Theme Update

### PR DESCRIPTION
**Pull request by Levelent**
A few small changes to drop the shield, switch out the logo/background, and make small improvements such as renaming one of the exec roles and editing a hard-coded url. Brings back the img folder to store most of the new assets in.

### CODE REVIEW

**Review by joshdavies14 — CHANGES_REQUESTED** [review]
Few comments on things, along with the following:
- `logo.svg` is used in places the CMS, so references to this file in `‎lib/templates/wagtailadmin/404.html` and `lib/templates/wagtailadmin/base.html` need to be updated to the new logo location.
- In the `sponsor_homepage` and `sponsor_sidebar` templates, worth noting that there might end up being merge conflicts down the line with changes I made in the dark-mode-fixes branch, but those changes are fine.
- PR should be merging into the `develop` branch, not `master`

File: static/scss/new/_settings_base.scss [hunk 1]
```diff
@@ -27,9 +27,9 @@ $container-offset: 128px;
 $footer-background-color: #2C3134;
 $footer-color: hsl(0, 0, 100%);
 
-$primary: #3d5afe;
+$primary: #3A7DFF;
```
[the remark this answers is not in the document]
Comment (Levelent) [reply static/scss/new/_settings_base.scss:30]: Updated the blue to 3A69FF to pass contrast checks against white - the previous yellow was already failing contrast checks, this one fails them less.

[the remark this answers is not in the document]
Comment (joshdavies14) [reply static/scss/new/_settings_base.scss:30]: `#3A69FF` still fails contrast checks against the two greys in use on the site: see https://webaim.org/resources/contrastchecker/?fcolor=3A69FF&bcolor=2C3134 and https://webaim.org/resources/contrastchecker/?fcolor=3A69FF&bcolor=2D3338

[the remark this answers is not in the document]
Comment (Levelent) [reply static/scss/new/_settings_base.scss:30]: So did the previous blue, if you check - we never use that colour combination for text, so this shouldn't be an issue. It even passes contrast checks on the greys more than 3D5AFE does, the original colour.

[the remark this answers is not in the document]
Comment (joshdavies14) [reply static/scss/new/_settings_base.scss]: I agree on the blue being better than before with the greys; however, commit 9e491f66eaae3c1f318c568f939dd158fc74b1d9 was written to not use the original blue against grey backgrounds (primarily in dark mode) to mitigate the accessibility issues. If you want to revert to the original/actual blue in dark mode (as indicated in #48), then the contrast checks should be passing against the greys too.

**Comment by Levelent** [comment]
Discussed on Discord, fine to merge now and and fix link colours later as a separate issue

**Pull request merged**